### PR TITLE
refactor: unite name

### PR DESCRIPTION
**Pull request by warmthsea**
Please make sure these boxes are checked before submitting your PR, thank you!

- [ ] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [ ] Make sure you are merging your commits to `dev` branch.
- [ ] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

File: .vscode/settings.json
Comment (warmthsea) [inline .vscode/settings.json:1]: main dev code

File: packages/components/select-v2/src/defaults.ts [hunk 1]
```diff
@@ -120,7 +125,8 @@ export const SelectProps = buildProps({
    */
   modelValue: {
     type: definePropType<
-      any[] | string | number | boolean | Record<string, any> | any
+      // eslint-disable-next-line prettier/prettier
```
Comment (warmthsea) [inline packages/components/select-v2/src/defaults.ts:128]: IDE code highlight warning

**Review by warmthsea — COMMENTED** [review]
Mainly to make the structure consistent with other components.

**Comment by warmthsea** [comment]
https://github.com/element-plus/element-plus/pull/16746

**Comment by makedopamine** [comment]
https://github.com/element-plus/element-plus/pull/16746#discussion_r1597627914

**Pull request closed without merging**